.github/workflows/cargo.yml

```diff
@@ -268,6 +255,50 @@ jobs:
           cat ./reports/ci-report.sarif
           diff ./reports/report.sarif ./reports/ci-report.sarif
 
+
+  third-party-reports:
+    name: Check 3rd party Reports
+    runs-on: ubuntu-latest
+    steps:
+      - name: foundry-toolchain
+        uses: foundry-rs/foundry-toolchain@v1.2.0
+
+      - name: Checkout sources
+        uses: actions/checkout@v2
```

Comment:
@TilakMaddy 

Can you:
- Merge dev into here
- Remove the change to this file

See if it passes?